src/lib/sessionStorage.ts

```diff
@@ -24,7 +25,8 @@ const sessionStorage = new AppStorage<{
   chatPositions: {
     [peerId_threadId: string]: ChatSavedPosition
   },
-  langPack: LangPackDifference
+  langPack: LangPackDifference,
+  drafts: AppDraftsManager['drafts']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
  drafts: AppDraftsManager["drafts"]
```